nucypher/crypto/powers.py

```diff
@@ -146,7 +146,7 @@ def account(self) -> str:
 
     def activate(self, password: str = None):
         """Be Consumed"""
-        self.blockchain.connect(sync_now=False)
+        self.blockchain.connect(fetch_registry=True, sync_now=False)
```

Comment:
Since `False` is the default now, this may not be necessary (depending on whether the intention here is "do the default action" or "never sync")

Reply:
`fetch_registry=True` may clash with a fix for #1166. Is it truly necessary to always fetch registry here?

Reply:
I think this can be handled elsewhere, no critical reason to be here other than being prudent.